### ISSUE TITLE
ScrollPane ScrollPosition no Setter

### ISSUE THREAD

**Issue by captkirk88** [MyraUI/Myra#23]
I created a Console (i.e Panel with ScrollPane, TextBlock and TextField) and I am unable to automatically set the scroll position to the very bottom.  ScrollPane's ScrollPosition only has a Getter.  You can ResetScroll but why not make it so that the scroll position can be set?

Would you like me to submit a pull request?

**Comment by rds1983** [comment]
Sure, thanks in advance!

**Comment by captkirk88** [comment]
Pull request ain't gonna happen.  It keeps adding the bin/ folders and it is going to cause conflicts with your master branch.  And I am not too good with git enough to know why it keeps automagically adding those folders so I will work on the ScrollPane file itself then post a gist to it when I update it.

**Comment by captkirk88** [comment]
Actually after reviewing it, seems like a very small change would enable me to at least work with it.

Change
```csharp
		public Point ScrollPosition
		{
			get { return _scrollPosition; }
		}
```
To
```csharp
		public Point ScrollPosition
		{
			get { return _scrollPosition; }
                        set { _scrollPosition = value; }
		}
```

**Comment by rds1983** [comment]
Yeah, when I fight with git, the git usually wins too :)
Thanks for that code, I'll make sure it'll be in the code base.

**Comment by rds1983** [comment]
Fixed in the version 0.5.6

**Comment by captkirk88** [comment]
Thank you!
